ts/sagas/startup/loadSessionInformationSaga.ts

```diff
@@ -21,15 +23,20 @@ import { SagaCallReturnType } from "../../types/utils";
  *        a saga.
  */
 export function* loadSessionInformationSaga(
-  getSession: TypeofApiCall<GetSessionStateT>
+  getSession: ReturnType<typeof BackendClient>["getSession"]
```

Comment:
@Undermaken note that since we changed signature of `getSession`, it's type can't be automatically generated from `GetSessionStateT`, we should instead refer to the type returned by `BackendClient`